tests/test_diaSourceMetrics.py

```diff
@@ -30,17 +30,17 @@ class DiaSourceTest(lsst.utils.tests.TestCase):
     and a plot is generated without falling over."""
 
     def setUp(self):
-        testFile = "tests/assocDiaSrc_test.ecsv"
+        testFile = "tests/data/assocDiaSrc_test.ecsv"
         self.data = Table.read(testFile)
 
     def test_metrics(self):
         """Test that metrics have the expected values from the test data."""
         NumDiaSources = diaAtool.NumGoodDiaSourcesMetrics()
         NumDiaSources.finalize()
         DiaSourceCount = NumDiaSources(self.data)
-        self.assertEqual(DiaSourceCount["numAllDiaSources"].quantity.value, 558)
-        self.assertEqual(DiaSourceCount["numGoodDiaSources"].quantity.value, 547)
-        self.assertEqual(DiaSourceCount["ratioGoodToAllDiaSources"].quantity.value, 547 / 558)
+        self.assertEqual(DiaSourceCount["numAllDiaSources"].quantity.value, 2994)
```

Comment:
This seems like a big change - any idea why?

Reply:
Yes - I didn't know the data ID that was used to generate the original dataset, so I guessed my own. This means that the output catalog has grown in size from the prior 558 entries to 2994. Undoubtedly, my data ID is not the same one that was used to generate the original test sample.

Reply:
OK that is fair! Whoever made that original dataset should have documented it better 😜 would you consider adding the script or similar you used to generate the test data for next time?

Reply:
I added my workflow into this Jira ticket comment: https://rubinobs.atlassian.net/browse/DM-49274?focusedCommentId=581302.

Is that a sufficient place to document this @mrawls?

Reply:
That is great - my initial thought was a text file or script of some kind in the new data subdirectory, but honestly having it on the Jira ticket is at least as good, thank you